commands/server/start.go

```diff
@@ -2,29 +2,41 @@ package server
 
 import (
 	"flag"
+	"fmt"
 
-	"github.com/pkg/errors"
-
+	"github.com/iov-one/weave/coin"
 	"github.com/tendermint/tendermint/abci/server"
 	abci "github.com/tendermint/tendermint/abci/types"
 	cmn "github.com/tendermint/tendermint/libs/common"
 	"github.com/tendermint/tendermint/libs/log"
 )
 
 const (
-	flagBind  = "bind"
-	flagDebug = "debug"
+	flagBind   = "bind"
+	flagDebug  = "debug"
+	flagMinFee = "min_fee"
 )
 
-func parseFlags(args []string) (string, bool, error) {
+func parseFlags(args []string) (string, bool, coin.Coin, error) {
 	// parse flagBind and return the result
 	var addr string
 	var debug bool
+	var minFeeStr string
+	minFee := coin.Coin{}
+
 	startFlags := flag.NewFlagSet("start", flag.ExitOnError)
 	startFlags.StringVar(&addr, flagBind, "tcp://localhost:46658", "address server listens on")
+	startFlags.StringVar(&minFeeStr, flagMinFee, "0 IOV", "minimal anti-spam fee")
 	startFlags.BoolVar(&debug, flagDebug, false, "call stack returned on error")
 	err := startFlags.Parse(args)
-	return addr, debug, err
+
+	if err != nil {
+		return addr, debug, minFee, err
+	}
+
+	err = minFee.UnmarshalJSON([]byte(minFeeStr))
```

Comment:
This will fail. The default must be `"\"0 IOV\""`. Ugly huh?
Rather than requiring ugly command line args, just add some `"` on both sides of minFeeStr before parsing